check/testdata/valid-legacy-files/2.0-guide.html.markdown

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Can you please add this test file case in `check/legacy_guide_file_test.go`? We should probably also cover this in the registry checks as well since it is supported. Thanks!